test/livebook_web/utils_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule LivebookWeb.UtilsTest do
+  use ExUnit.Case, async: true
+
+  alias LivebookWeb.Utils
+
+  describe "name_to_id/1" do
+    test "Empty string", do: assert(Utils.name_to_id("") == "")
```

Comment:
We should default to an arbitrary string, because empty is not really much of an id. Perhaps "untitled"?

Reply:
Yeah, actually an empty string isn't a valid ID according to the HTML5 spec, so I've removed this example.  Probably Livebook should handle making sure that there's always a title at a higher level?  It seems to change it to "Untitled" already.

Reply:
Actually yeah, I thought we could drop some characters and that would end up in an empty html id, but since we only replace white space with `-` it's never the case.